docs/spec/deprecated-schema-v1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Update deprecated schema v1 images
```

Comment:
Should we make this `Update deprecated image manifest version 2, schema 1 images to schema 2` ?